cloudbuild.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019 The Forseti Security Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This defines the build steps of Forseti docker images. The google container
+# builder must be invoked against the root of the git directory.
+#
+
+steps:
```

Comment:
Is there a reason we can't reuse the shell scripts we have for some of the below tasks vs the explicit commands here? Trying to reduce duplication of effort if/when any of this changes if possible.

  - ./install/scripts/docker_run_forseti.sh
  - ./install/scripts/docker_unittest_forseti.sh
  - ./install/scripts/docker_pystyle_forseti.sh